railties/test/generators/actions_test.rb

```diff
@@ -265,9 +266,10 @@ def test_initializer_should_write_date_to_file_with_block_in_config_initializers
   end
 
   def test_generate_should_run_script_generate_with_argument_and_options
-    assert_called_with(generator, :run_ruby_script, ["bin/rails generate model MyModel", verbose: false]) do
+    assert_called(Rails::Generators::ModelGenerator, :source_root, times: 0, returns: destination_root) do
```

Comment:
If we want to test this we should not be stubbing anymore, this test give no guarantees this will work.

Reply:
@rafaelfranca agreed, I don't believe this assertion is every useful...would you be OK with me replacing the test to be this?

```ruby
action :generate, "model", "MyModel"
assert_file('app/models/my_model.rb')
```

That way we're actually testing the results of the operation rather than whether some command was called...

Reply:
Yes.